src/Symfony/Component/Workflow/Event/Event.php

```diff
@@ -60,7 +60,7 @@ public function getWorkflowName()
         return $this->workflow->getName();
     }
 
-    public function getMetadata(string $key, object $subject)
+    public function getMetadata(string $key, ?object $subject)
```

Comment:
Actually, the type hint should be removed.

https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Workflow/Metadata/MetadataStoreInterface.php#L34-L37

Reply:
Thanks, pushed a commit to remove it.